src/explorer.ts

```diff
@@ -319,13 +323,13 @@ export class OpenShiftExplorer implements TreeDataProvider<ExplorerItem>, Dispos
             pod.status.containerStatuses.forEach((cs) => {
                 if (cs.state?.waiting) {
                     const reason = cs.state.waiting.reason;
-                    const message = cs.state.waiting.message;
+                    const message = this.stripTimestamp(cs.state.waiting.message);
 
                     inCrashLoopBackOff = inCrashLoopBackOff || reason === 'CrashLoopBackOff';
 
                     const msg = `${reason}: ${message ? message.trim(): 'No valuable message'}`;
                     // Skip duplicates and show not more than 10 errors
```

Comment:
Update or remove this comment if it's stale

Reply:
ops.